index.html

```diff
@@ -5,40 +5,265 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css"/>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <header>
-        <a class="logo" href="#">My Logo</a>
-        
-            <nav class="menu_icon"><img src="images/Union.png"/></nav>
-                  
-    </header>
     <section>
-        <div class="headline">
-            <main>
-                <h1>
-                     I’m May Pyone.
-                       Glad to see you!
-                </h1>
-                
+    
+        <header class="taskbar">
+            <a class="logo" href="#"><strong>My Logo</strong></a>
+            
+                <nav class="menu_icon"><img src="images/Union.png"/></nav>
+                      
+        </header>
+        <section class="inner1">
+            <div class="intro_one">
+                    <h1>
+                         I’m May Pyone.
+                           Glad to see<br> you!
+                    </h1>
+                    
+                    <p>
+                        I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+                    <p class="connect">LET'S CONNECT</p>
+                 <ul class="connect_menu">
+                    <li><a href="#"><img src="images/twitter.png"/></a></li>
+                    <li><a href="#"><img src="images/linkin.png"/></a></li>
+                    <li><a href="#"><img src="images/M.png"></a></li>
+                    <li><a href="#"><img src="images/git.png"></a></li>
+                    <li><a href="#"><img src="images/fingers.svg"></a></li>
+                 </ul>
+                            
+                  
+            </div>
+            
+        </section>
+
+
+
+
+
+    <div class="grid_container">
```

Comment:
- [optional] please kindly put the works section in a section tag as well and not a div like i explained for the about section. nice work.